detox/package.json

```diff
@@ -41,6 +41,7 @@
     "@babel/core": "^7.4.5",
     "bunyan": "^1.8.12",
     "bunyan-debug-stream": "^1.1.0",
+    "callsites": "^3.1.0",
```

Comment:
this package is 5 lines of code, what if you paste them into detox?

Reply:
done :)